package.json

```diff
@@ -155,34 +155,34 @@
     "jsdom-global": "3.0.2",
     "lodash.merge": "4.6.2",
     "merge-stream": "2.0.0",
-    "mocha": "10.4.0",
+    "mocha": "10.7.0",
     "mq-polyfill": "1.1.8",
     "normalize.css": "8.0.1",
     "path": "0.12.7",
-    "postcss": "8.4.38",
+    "postcss": "8.4.40",
     "postcss-csso": "6.0.1",
     "postcss-discard-comments": "6.0.2",
     "postcss-import": "15.1.0",
     "postcss-loader": "7.3.3",
-    "postcss-preset-env": "9.5.11",
-    "prettier": "3.2.5",
+    "postcss-preset-env": "9.6.0",
+    "prettier": "3.3.3",
     "react-dom": "17.0.2",
     "resolve-url-loader": "5.0.0",
-    "sass": "1.77.0",
-    "sass-embedded": "1.77.0",
+    "sass": "1.77.6",
+    "sass-embedded": "1.77.5",
     "sass-loader": "13.3.2",
     "sass-true": "6.0.1",
     "sinon": "12.0.1",
-    "snyk": "1.1291.0",
+    "snyk": "1.1292.2",
     "style-loader": "3.3.3",
-    "stylelint": "16.5.0",
+    "stylelint": "16.8.1",
```

Comment:
**issue**: This task is failing locally.

```sh
npm run lint:sass
```

Results in an error:
```sh

> @uswds/uswds@3.8.1 lint:sass
> gulp lintSass

[15:16:42] Using gulpfile ~/web/uswds/gulpfile.js
[15:16:42] Starting 'lintSass'...
(node:13738) [stylelint:002] DeprecationWarning: The CommonJS Node.js API is deprecated.
See https://stylelint.io/migration-guide/to-16
(Use `node --trace-deprecation ...` to show where the warning was created)
[15:16:53] Finished 'lintSass' after 11 s
```

---

This also appears, but doesn't block `npx gulp buildSass`.

Reply:
**note:** This is just a deprecation _warning_. It was introduced in #5782 and flagged in [this comment](https://github.com/uswds/uswds/pull/5782#pullrequestreview-2043394195)

> **Stylelint CommonJS Node.js API is deprecated**
> This isn't an issue for this version of Stylelint (`16.5.0`) but will prevent us from updating to the next major version unless we migrate to ESM. [^1]

[^1]: [Stylint: Deprecated CommonJS API](https://stylelint.io/migration-guide/to-16/#deprecated-commonjs-api)

If you introduce a sass lint error it'll appropriately flag it.

![image](https://github.com/user-attachments/assets/77e012ff-ab8c-4d73-92d1-8f6de0620a7b)

Reply:
Thanks @mahoneycm. Is there an issue for it so we can triage? Seems urgent.

Reply:
We have #4986 which is broken into sub-tasks. I've added [a comment](https://github.com/uswds/uswds/issues/4986#issuecomment-2299030609) outlining the need to migrate to ESM before the next major version of Stylelint is released.

I'll add a discussion item to a team sync and we can discuss the ESM migration and where it fits into our roadmap.